codebase/superdarn/src.bin/tk/tool/map_addhmb.1.17/map_addhmb.c

```diff
@@ -295,7 +299,8 @@ int main(int argc,char *argv[])
 
         for (j=0; j<nlat; j++) latcnt[j] = 0;
         for (i=0; i<grd[buf]->vcnum; i++) {
-          if (fabs(grd[buf]->data[i].vel.median) < vel_min) continue;
+        
+          if (fabs(grd[buf]->data[i].vel.median) < vel_min || grd[buf]->data[i].wdt.median < wdt_min) continue;
```

Comment:
```suggestion
          if ( (fabs(grd[buf]->data[i].vel.median) < vel_min) ||
               ((grd[buf]->xtd) && (grd[buf]->data[i].wdt.median < wdt_min)) ) continue;
```

@pasha-ponomarenko would something like this solve the present issue?  It seems like requiring the calculation of spectral width values for all grid files is a more significant change than what is being proposed in this pull request.

(note there may also need to be a `grd->xtd=0;` inserted between lines 352-353 of `readmap.c` to ensure the `xtd` flag is being properly set)

Reply:
Thanks @egthomas , it looks fine for me!